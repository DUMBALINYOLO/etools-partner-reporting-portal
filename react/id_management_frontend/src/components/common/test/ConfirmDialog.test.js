import React from 'react';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import ConfirmDialog from '../ConfirmDialog';

describe('ConfirmDialog component', () => {
    it('passes', () => {
        const open = true;
        const onClose = jest.fn();
        const title = 'hello';
        const message = 'some message';
        const onConfirm = jest.fn();

        const wrapper = shallow(<ConfirmDialog
            open={open}
            onClose={onClose}
            title={title}
            message={message}
            onConfirm={onConfirm}
        />);

        expect(toJSON(wrapper)).toMatchSnapshot();
        expect(wrapper.shallow().length).toBe(1);
    });
});