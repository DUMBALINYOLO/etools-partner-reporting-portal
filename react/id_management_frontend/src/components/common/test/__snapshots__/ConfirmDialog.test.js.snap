// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmDialog component passes 1`] = `
<ConfirmDialog
  classes={
    Object {
      "text": "ConfirmDialog-text-1",
    }
  }
  message="some message"
  onClose={[Function]}
  onConfirm={[Function]}
  open={true}
  title="hello"
/>
`;
